storage/remote/generic/generic.proto

```diff
@@ -27,10 +26,9 @@ message LabelPair {
 }
 
 message TimeSeries {
-  string name = 1;
-  repeated LabelPair labels = 2;
```

Comment:
Don't think that's necessary for anyone? But we can still add it later if some use case needs it - if we do it now and guarantee it, then we can't remove it again.